packages/common/src/types/schema.ts

```diff
@@ -58,6 +58,24 @@ export type AddFromMasterListInput = {
 
 export type AddFromMasterListResponse = AddFromMasterListError | RequisitionLineConnector;
 
+export type AllocateOutboundShipmentUnallocatedLineError = {
```

Comment:
all changes from the gql codegen. picking up changes made to the API and merged into the api develop branch